StatBot/Database/PostgresRepositories/PostgresUserRepository.cs

```diff
@@ -33,7 +33,7 @@ public void Delete(int id)
         public void Dispose()
         {
             _context.Dispose();
-            GC.SuppressFinalize(this);
+            //GC.SuppressFinalize(this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS125)